playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Technology choices
+description: How to write technical plans at Artsy
+---
+
+# Technical planning
+
+Technical planning is a crucial part of our engineering process. Writing tech plans helps us:
+- Keep track of technical decisions that we've made, and the associated rationale.
+- Streamline a team's development of a feature and align within and across teams.
+- Communicate and collaborate effectively on technical decisions in an async manner.
+- Think through problems in a way that allows developers to practice technical decision-making & grow.
+- Get feedback early on in the development process, to avoid costly U-turns further down the line.
+
+Developers should use their judgment when deciding whether or not to write a tech plan, but we recommend them when planning:
+1. Technical design for medium to large features
+1. Architectural changes or new design patterns
+1. Changes affecting other teams' products or engineers' workflows
+1. New integrations or services
+
+If the feature you're working on follows our best-practices and doesn't require technical decisions larger than what would fit in a single PR, it's OK to skip.
+
+If you're working on a small feature or it doesn't fit the criteria above but you still want to write one, that's fine! Feel free to keep it small.
+
+## Tech Plans vs. Product Briefs
+Technical plans discuss and propose a technical implementation, while product briefs should govern the design, user experience, and business requirements of a feature. We recommend linking to an associated product brief for context, but product and requirement decisions should live on the product brief, not on the tech plan.
```

Comment:
Should we clarify when to start a tech plan? Should we wait until there's a formal (and signed-off) product brief? Or (as in the paragraph below) do they tend to work symbiotically? To have a fully spec'd-out product brief seems ideal, but at times it doesn't seem entirely practical if the target seems to be moving. 

Is there a situation where a technical plan is written _without_ a product brief? And if so, when?

Reply:
1. In my experience tech plans are easiest to write and review when the desired outcome is clear. Sometimes the desired outcome of a project brief is clear before it receives sign-off, and sometimes it's not clear even after it receives sign-off. I would avoid creating a dependency between technical plans and project briefs based on the state of a project brief for this reason. I think that the following paragraphs do a good job of explaining how engineers should react in a case where the desired outcome is unclear.
2. I think that tech plans should originate from RFCs, practice meetings, future fridays, etc. as well as from product briefs.

Reply:
+1 to what @iskounen said!